docs/source/advancedQueries.rst

```diff
@@ -192,15 +168,16 @@ Let's search our data for the event file of **sub-009** for the **deduction** ta
     print(events_sub009_deduc)
     #Output
     #['./ancp-bids/tests/data/ds003483/sub-009/ses-1/meg/sub-009_ses-1_task-deduction_run-1_events.tsv']
-Note, if your BIDS dataset contains metadata for your event files you can specify if you want to search
-for the metadata or the actual event files by setting the extension parameter to '.json' or '.tsv', respectively.
+Note, if your BIDS dataset contains metadata for your event files (i.e. a file describing the columns of your events file)
+you can specify if you want to search for the metadata or the actual event files
+by setting the extension parameter to '.json' or '.tsv', respectively.
 
 We can search our data for the other files described above by setting the extension value to one
 of the values defined above.
 
-Moreover, the ancpbids library offers convenience functions to query for (or access?)
-the dataset_description.json (and the participants.tsv which are the most common metadata
-from the first level of hierarchy within the dataset, i.e.metadata that describes the whole dataset.)
+Moreover, the ancpbids library offers a convenience function to query for (or access?)
```

Comment:
both are correct: you can query for a dataset_description file starting from he root folder or just access the field directly